builder/internals.go

```diff
@@ -316,16 +358,29 @@ func calcCopyInfo(b *Builder, cmdName string, cInfos *[]*copyInfo, origPath stri
 			}
 		}
 
-		if err := system.UtimesNano(tmpFileName, times); err != nil {
-			return err
+		// UtimesNano is not supported on Windows
+		if runtime.GOOS != "windows" {
+			if err := system.UtimesNano(tmpFileName, times); err != nil {
+				return err
+			}
 		}
 
 		ci.origPath = filepath.Join(filepath.Base(tmpDirName), filepath.Base(tmpFileName))
 
+		// Assert that ci.destPath is in Linux format, so contains no back-slashes
+		assert.Assert(!strings.Contains(ci.destPath, "\\"), "internals.go::calcCopyPath() ci.destPath contains a \\ - "+ci.destPath)
```

Comment:
why are we doing this instead of just returning an error?
I don't think we should ever panic due to user-provided data.  I'm not all that thrilled with logging it either, but that's a different story.